backend/backend/Routes/Lost_and_Found/lost.py

```diff
@@ -22,7 +22,7 @@ async def add_item( request: Request,
         form_data_dict = json.loads(form_data)
         user_id = get_user_id(request)
         with conn.cursor() as cur:
-            cur.execute( insert_in_lost_table( form_data_dict, user_id ) )
+            cur.execute( insert_in_table(0, form_data_dict, user_id ) )
```

Comment:
You were supposed to pass a `TableType.LOST` or `TableType.FOUND`, but you have passed integers, and it gives errors. Fix this error in all the  files.